src/glang/graphene_parser.py

```diff
@@ -503,9 +515,7 @@ def UnaryOperatorUse(self, node: lp.UnaryOperatorUse) -> FlattenedExpression:
 
         flattened_expr = FlattenedExpression(rhs.subexpressions)
         call_expr = self._program.lookup_call_expression(
-            node.name,
-            [],  # Don't specialize operators
-            [rhs.expression()],
+            node.name, [], [rhs.expression()], node.meta  # Don't specialize operators
```

Comment:
This comment should probably be deleted now it's moved, I don't think its very useful anymore.

Reply:
Oops, I just missed the trailing comma and black re-arranged everything. I fixed it now.